### PR TITLE
Add submission types and deadlines

### PR DESCRIPTION
**Pull request by csoneson**
@kevinrue, @lwaldron - thoughts before merging this?

### CODE REVIEW

File: content/submissions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Important dates
+
+* Jan 22, 2021: Call for abstracts opens
+* Mar 9, 2021: Abstract submission closes
+* April 16, 2021: Notification of decision
+
+## Submission types
+
+### Short talk
+Focused on a package, an application note, or the presentation of a new research project that seeks input and collaborations from the community.
```
Comment (lwaldron) [inline content/submissions.md:15]: Maybe, "or a presentation of current research. Presentations seeking collaboration or input from the community are encouraged." Only making this suggestion not to eliminate recently completed research?

File: content/submissions.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### Software demo ("short workshop")
+30-45 min demonstration of a software package or workflow.
+
+### Long workshop
+1.5 - 2 hour interactive workshop, where participants will be expected to have the time and opportunity to follow along and perform an analysis themselves.
+
+### Poster
```
Comment (lwaldron) [inline content/submissions.md:23]: ### Digital Poster

File: content/submissions.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Posters can be submitted and displayed/presented as plain pdf posters, shiny apps, web pages, ... be creative! Posters will be presented in a dedicated remote session.
```
Comment (lwaldron) [inline content/submissions.md:24]: Digital posters can be...

File: content/submissions.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+### Birds-of-a-feather session
+Free-format, interactive discussion session on a topic relevant to the Bioconductor community.
```
Comment (lwaldron) [inline content/submissions.md:27]: Free-format, organizer-hosted interactive discussion session...

**Review by lwaldron — COMMENTED** [review]
Some minor suggestions inline, looks good!

File: content/submissions.md (hunk 1 above)
Comment (csoneson) [reply content/submissions.md:15]: Yes, good point

File: content/submissions.md (hunk 2 above)
Comment (csoneson) [reply content/submissions.md]: Fixed

File: content/submissions.md (hunk 3 above)
Comment (csoneson) [reply content/submissions.md]: Fixed

File: content/submissions.md (hunk 4 above)
Comment (csoneson) [reply content/submissions.md]: I changed to "organizer-lead", just to not indicate that the organizer has to host the session on another platform or so.

**Review by mdozmorov — APPROVED** [review]

**Comment by mdozmorov** [comment]
Will also add the dates to the front page

File: content/submissions.md (hunk 4 above)
Comment (lwaldron) [reply content/submissions.md]: perfect.

**Pull request merged**